packages/es-client/indexer.js

```diff
@@ -288,11 +288,13 @@ async function indexGranule(esClient, payload, index = defaultIndexAlias, type =
  * Upserts a granule in Elasticsearch
  *
  * @param {Object} params
- * @param {Object} params.esClient - Elasticsearch Connection object
- * @param {Object} params.updates  - Updates to make
- * @param {string} params.index    - Elasticsearch index alias (default defined in search.js)
- * @param {string} params.type     - Elasticsearch type (default: granule)
- * @param {string} [params.refresh] - whether to refresh the index on update or not
+ * @param {Object} params.esClient        - Elasticsearch Connection object
+ * @param {Object} params.updates         - Updates to make
+ * @param {string} params.index           - Elasticsearch index alias (default defined in search.js)
+ * @param {string} params.type            - Elasticsearch type (default: granule)
+ * @param {string} [params.refresh]       - whether to refresh the index on update or not
+ * @param {bool}   writeConstraints       - boolean toggle restricting if conditionals should
```

Comment:
Is there a difference between `bool` and `boolean`?

Reply:
Sorry, that's just other-language-shorthand.  That should properly be boolean, I'm surprised it didn't lint fail 😓

Reply:
fb29cb5f